lib/std/debug.zig

```diff
@@ -441,7 +441,12 @@ pub fn dumpStackTraceFromBase(context: *ThreadContext) void {
 
         var it = StackIterator.initWithContext(null, debug_info, context) catch return;
         defer it.deinit();
-        printSourceAtAddress(debug_info, stderr, it.unwind_state.?.dwarf_context.pc, tty_config) catch return;
+
+        const pc_addr = if (builtin.target.os.tag.isDarwin() and native_arch == .aarch64)
```

Comment:
This should at least have a comment referring to the other comment that explains why this is necessary.

Reply:
Sure, done.